admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
```

Comment:
There's a module called Memcache, yet we still call these things memcaches. I don't see the problem with using de-facto common terminology?